test/onnx/test_operators.py

```diff
@@ -55,6 +55,11 @@ def forward(self, *args):
 
 class TestOperators(TestCase):
 
+    def skipAlwaysForCI(func):
```

Comment:
If we define this wrapper like the following, why not directly using unittest.SkipTest :-)

How about https://github.com/pytorch/pytorch/blob/cca247635c6edb323176eeac7a18d3e9ab71c558/test/onnx/test_models.py#L80

Reply:
great, never know such thing exist